omod/src/main/java/org/openmrs/module/operationtheater/web/resource/SurgicalAppointmentResource.java

```diff
@@ -193,4 +201,20 @@ public static void setAttributes(SurgicalAppointment surgicalAppointment, Set<Su
 		}
 		surgicalAppointment.setSurgicalAppointmentAttributes(attrs);
 	}
+	
+	@PropertyGetter("patientObservations")
+	public static List<Obs> getPatientObservations(SurgicalAppointment surgicalAppointment) {
+		String obsConceptNames = Context.getAdministrationService().getGlobalProperty("obs.conceptsForOT");
```

Comment:
@kavitha-sundararajan can we have global property added with sample concept name or with null value instead of not adding the global property

Reply:
It makes perfect sense for the module to create the global property with a full description that includes an example.

Reply:
Is there a reason why you are using concept names instead of concept uuid or mappings?

Reply:
@dkayiwa  The reason we are using concept names is whenever the db is changed or in different implementation, the uuid might get changed each time but the concept names would remain same. So we went with concept names. I am not sure about concept mappings. Could you add some details on how it can be done with concept mappings ?

Reply:
As suggested, created global property with description in the module. Refactored reading the concepts by using concept mappings instead of concept names. Please have a look